install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -142,10 +142,10 @@ data:
           valueFrom:
             fieldRef:
               fieldPath: status.podIP
-        - name: ISTIO_META_POD_NAME
```

Comment:
Why ? It's backward incompatible and people may use it ( pilot is not the only consumer ).

The POD_NAME is important for other purposes, we still plan to move away from IP-as-key.

Reply:
we have POD_NAME already.. ISTIO_META_POD_NAME is not used anywhere in code/config. It was meant for pilot though never used..